bundle/manifests/node-healthcheck-operator.clusterserviceversion.yaml

```diff
@@ -406,7 +406,13 @@ spec:
                 ports:
                 - containerPort: 8443
                   name: https
-                resources: {}
+                resources:
```

Comment:
how did you decide for these limits?

Reply:
copy paste from another operator 🤷🏼‍♂️ 😉